src/js/content.js

```diff
@@ -3,69 +3,71 @@ import "../img/downvote.png";
 import "../img/upvote.png";
 
 function youtube_parser(url) {
-  var regExp = /^.*((youtu.be\/)|(v\/)|(\/u\/\w\/)|(embed\/)|(watch\?))\??v?=?([^#&?]*).*/;
-  var match = url.match(regExp);
-  return match && match[7].length == 11 ? match[7] : false;
+    var regExp = /^.*((youtu.be\/)|(v\/)|(\/u\/\w\/)|(embed\/)|(watch\?))\??v?=?([^#&?]*).*/;
+    var match = url.match(regExp);
+    return match && match[7].length == 11 ? match[7] : false;
 }
 
-chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
-  console.log("page change");
+chrome.runtime.onMessage.addListener((_request, _sender, _sendResponse) => {
+    console.log("page change");
 
-  setTimeout(() => {
-    init();
-  }, 500);
+    setTimeout(() => {
+        init();
+    }, 500);
 });
 
 function elementLoaded(el, cb) {
-  if ($(el).length) {
-    // Element is now loaded.
-    cb($(el));
-  } else {
-    // Repeat every 500ms.
-    setTimeout(() => {
-      elementLoaded(el, cb);
-    }, 500);
-  }
+    if ($(el).length) {
+        // Element is now loaded.
+        cb($(el));
+    } else {
+        // Repeat every 500ms.
+        setTimeout(() => {
+            elementLoaded(el, cb);
+        }, 500);
+    }
 }
 
 function init() {
-  var url = window.location.href;
-  var video_id = youtube_parser(url);
-  console.log(video_id);
-
-  elementLoaded(".ytd-video-primary-info-renderer", (el) => {
-    console.log("We're on a video!");
-
-    fetch("https://twitchtos.herokuapp.com/getrating?video_id=" + video_id)
-      .then((r) => r.text())
-      .then((result) => {
-        console.log(result);
-        if (parseInt(result) > 0 && parseInt(result) < 3) {
-          var scoretxt = "Decent";
-        }
-        if (parseInt(result) < 0 && parseInt(result) > -3) {
-          var scoretxt = "Bad";
-        }
-        if (parseInt(result) < -3) {
-          var scoretxt = "Terrible";
-        }
-        if (parseInt(result) > 3) {
-          var scoretxt = "Good";
-        }
-
-        var el = document.getElementsByClassName(
-          "ytd-video-primary-info-renderer"
-        )[0].children[5].children[1];
-        console.log(
-          document.getElementsByClassName("ytd-video-primary-info-renderer")[0]
-            .children[5]
-        );
-        $(".toscontainer").empty();
-        $(".toscontainer").remove();
-        el.insertAdjacentHTML("beforeend", '<div class="toscontainer"></div>');
-        if (result == "0") {
-          $(".toscontainer").append(
-            `
+    var url = window.location.href;
+    var video_id = youtube_parser(url);
+    console.log(video_id);
+
+    elementLoaded(".ytd-video-primary-info-renderer", (_el) => {
+        console.log("We're on a video!");
+
+        fetch("https://twitchtos.herokuapp.com/getrating?video_id=" + video_id)
+            .then((r) => r.text())
+            .then((result) => {
+                console.log(result);
+                let scoretxt;
```

Comment:
I think this behaves the same.  
The old code worked like this:

 1. If cond1, then scoretxt = value
 2. If cond2, then scoretxt = otherValue
Etc...
 3. Use scoretxt, which will be undefined if none of the conditions are true.

New code works like this:
 1. Declare scoretxt as undefined
 2. If cond1, then scoretxt = value
 3. Else if cond2, then scoretxt = otherValue
Etc...
 4. Use scoretxt, which will be undefined if none of the conditions were true